scripts/go-lint-all.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -e -o pipefail
+set -e
```

Comment:
I am removing pipefail because none of the pipe logic seems to require that a non-terminal expression should produce a global error; the existing case (ln 45) was a false negative.